tools-v2/pkg/config/bs.go

```diff
@@ -47,6 +47,12 @@ const (
 	CURVEBS_PASSWORD           = "password"
 	VIPER_CURVEBS_PASSWORD     = "curvebs.root.password"
 	CURVEBS_DEFAULT_PASSWORD   = "root_password"
+	CURVEBS_USERNAME           = "username"
```

Comment:
user and username should be one, maybe merge it?

Reply:
done